setup.py

```diff
@@ -109,6 +114,9 @@ def run(self):
                   '"protobuf-compiler" and "libprotobuf-dev" packages.')
           elif sys.platform == 'darwin':
             print('On Mac, protobuf is often installed via homebrew.')
+          else:
+            print('On Windows, protoc should be installed and added '
+                  'to the path.')
```

Comment:
Maybe you could leave more of a bread crumb here for how the user should go about getting protoc for windows, even if it's just to point them at the protobuf website. The Linux and Mac strings at least mention which method is recommended to get protoc.

Reply:
Added a link to https://github.com/google/protobuf/releases